### ISSUE TITLE
No file named model.h5 

### ISSUE THREAD

**Issue by nehalb123** [naokishibuya/car-behavioral-cloning#15]
I trained the model and for some epochs it generated model-00x.h5 files **but there was no file named model.h5.**
Why is it so?

**Comment by naokishibuya** [comment]
Hi @nehalb123 ,
The script (model.py) generates a model file with epoch number when the validation loss improves.  

```python
    checkpoint = ModelCheckpoint('model-{epoch:03d}.h5',
                                 monitor='val_loss',
                                 verbose=0,
                                 save_best_only=args.save_best_only,
                                 mode='auto')
```

Later, you can copy or rename a model-xxx.h5 to model.h5 for testing.
